cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -17,6 +17,7 @@ metadata:
   name: rook-ceph # namespace:cluster
 # OLM: BEGIN OBJECTBUCKET ROLEBINDING
 ---
```

Comment:
RBAC changes are needed in the helm chart, unfortunately they are still duplicated from common.yaml

Reply:
🤦‍♀️  I'll update that

I wonder if its worth generating common.yaml from the helm charts.... 
Related, we can generate RBAC with kubebuilder the same way we do with CRDs, if we ever want to do that.

Reply:
One source of RBAC is certainly desired. Another complication is that the OLM is generated from common.yaml, so we would also need to generate that at the same time if we change that.

Reply:
On that note, a question for @leseb : is it necessary to separate all the components with things like `# OLM: BEGIN OBJECTBUCKET ROLEBINDING`? What is the purpose of those?

Reply:
Those delimiters are used for the olm generation [here](https://github.com/rook/rook/blob/master/cluster/olm/ceph/generate-rook-csv.sh#L161-L207).

Reply:
The delimiters are only used in that script itself. If we rewrite that script to generate the OLM differently from the helm chart, then we wouldn't need them.